package/src/components/Channel/Channel.tsx

```diff
@@ -1561,7 +1561,7 @@ const ChannelWithContext = <
           attachment.image_url = uploadResponse.file;
           delete attachment.originalFile;
 
-          dbApi.updateMessage({
+          await dbApi.updateMessage({
```

Comment:
Does not need to happen in this PR, but let's please seriously consider not `await`ing DB writes. We:

- Don't need to do that, it can happen in a different JS task queue
- It would be more performant that way
- The JS thread would not be needlessly clogged (for customers that rely on animations that aren't on the UI thread for example)

Reply:
Just leaving this here so we don't forget :D